test/unit/command/test_converge.py

```diff
@@ -38,3 +41,28 @@ def test_execute(mocker, patched_logger_info, patched_ansible_converge,
     patched_ansible_converge.assert_called_once_with()
 
     assert config_instance.state.converged
+
+
+def test_ansible_args(mocker, patched_logger_info, patched_ansible_converge,
+                      patched_config_validate, config_instance):
+    # Use the click test runner and mocks to ensure that ansible_args given on
+    # the converge commandline get correcty passed to the Scenarios constructor
+
+    # Mock out Scenarios to prevent execute_cmdline_scenarios from doing
```

Comment:
I am for the "why" comments but typically not for the "what" comments because if the code changes then we might not change the comment. The test code seems simple enough to note require such comments? Just a thought, not religious on comment style, promise.

Reply:
I err on the side of verbosity, since I never know how much is obvious after spending the time to figure out where to bolt on the patch. If it's clear to you what's going on, then I agree it's fine to reduce the commentary. :)

Reply:
Alright, I left a comment in there with a slightly different comment. Nothing asserts against the Scenarios mock, so I think leaving a little note along the lines of "No really, you wanna keep this mock in place..." is a safe choice. Let me know if you think what's there now is better.